ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -655,10 +637,10 @@ typedef struct
  * @param[in] detinputptr Cursor pointer of input + byte-per-index * index (detection)
  * @param[in] result The object returned. (pointer to object)
  */
-#define _get_object_i_tflite(bb, index, boxprior, boxinputptr, detinputptr, result) \
+#define _get_object_i_tflite(bb, index, total_labels, boxprior, boxinputptr, detinputptr, result) \
```

Comment:
Need to change the line 635 as well. 

Q. Does this mean the number of labels?

Reply:
@wooksong Please check below update.
```
 * @param[in] total_labels The count of total labels. We can get this from input tensor info. (1st dimension of LABEL_SIZE:TFLITE_SSD_DETECTION_MAX:1:1)
```

ssd_mobilenet_v2_coco.tflite has below tensor info.
```
input[0] >> type:7, dim[3:300:300:1] video stream (RGB 300x300)
output[0] >> type:7, dim[4:1:1917:1] BOX_SIZE:1:DETECTION_MAX:1
output[1] >> type:7, dim[91:1917:1:1] LABEL_SIZE:DETECTION_MAX
```